2025-04-28 20:32:55,214 INFO    MainThread:19329 [wandb_init.py:setup_run_log_directory():724] Logging user logs to /home/jupyter/wandb/run-20250428_203255-qtio4tf3/logs/debug.log
2025-04-28 20:32:55,214 INFO    MainThread:19329 [wandb_init.py:setup_run_log_directory():725] Logging internal logs to /home/jupyter/wandb/run-20250428_203255-qtio4tf3/logs/debug-internal.log
2025-04-28 20:32:55,214 INFO    MainThread:19329 [wandb_init.py:monkeypatch_ipython():643] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fd537f9dd80>
2025-04-28 20:32:55,214 INFO    MainThread:19329 [wandb_init.py:init():852] calling init triggers
2025-04-28 20:32:55,215 INFO    MainThread:19329 [wandb_init.py:init():857] wandb.init called with sweep_config: {}
config: {'epochs': 5, 'batch_size': 128, 'lr': 0.001, 'dropout': 0.5186114517062034, '_wandb': {}}
2025-04-28 20:32:55,215 INFO    MainThread:19329 [wandb_init.py:init():893] starting backend
2025-04-28 20:32:55,215 INFO    MainThread:19329 [wandb_init.py:init():897] sending inform_init request
2025-04-28 20:32:55,217 INFO    MainThread:19329 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-28 20:32:55,217 INFO    MainThread:19329 [wandb_init.py:init():907] backend started and connected
2025-04-28 20:32:55,227 INFO    MainThread:19329 [wandb_run.py:_label_probe_notebook():1369] probe notebook
2025-04-28 20:32:55,227 INFO    MainThread:19329 [wandb_run.py:_label_probe_notebook():1379] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-04-28 20:32:55,227 INFO    MainThread:19329 [wandb_init.py:init():1002] updated telemetry
2025-04-28 20:32:55,228 INFO    MainThread:19329 [wandb_init.py:init():1026] communicating run to backend with 90.0 second timeout
2025-04-28 20:32:55,501 INFO    MainThread:19329 [wandb_init.py:init():1101] starting run threads in backend
2025-04-28 20:32:55,771 INFO    MainThread:19329 [wandb_run.py:_console_start():2566] atexit reg
2025-04-28 20:32:55,772 INFO    MainThread:19329 [wandb_run.py:_redirect():2414] redirect: wrap_raw
2025-04-28 20:32:55,772 INFO    MainThread:19329 [wandb_run.py:_redirect():2483] Wrapping output streams.
2025-04-28 20:32:55,772 INFO    MainThread:19329 [wandb_run.py:_redirect():2506] Redirects installed.
2025-04-28 20:32:55,773 INFO    MainThread:19329 [wandb_init.py:init():1147] run started, returning control to user process
2025-04-28 20:33:02,639 INFO    MainThread:19329 [wandb_run.py:_finish():2314] finishing run paul-mriganka-personal/pytorch-intro/qtio4tf3
2025-04-28 20:33:02,640 INFO    MainThread:19329 [jupyter.py:save_history():477] not saving jupyter history
2025-04-28 20:33:02,640 INFO    MainThread:19329 [jupyter.py:save_ipynb():401] not saving jupyter notebook
2025-04-28 20:33:02,640 INFO    MainThread:19329 [wandb_init.py:_jupyter_teardown():628] cleaning up jupyter logic
2025-04-28 20:33:02,640 INFO    MainThread:19329 [wandb_run.py:_atexit_cleanup():2531] got exitcode: 0
2025-04-28 20:33:02,640 INFO    MainThread:19329 [wandb_run.py:_restore():2513] restore
2025-04-28 20:33:02,641 INFO    MainThread:19329 [wandb_run.py:_restore():2519] restore done
2025-04-28 20:33:04,092 INFO    MainThread:19329 [wandb_run.py:_footer_history_summary_info():4160] rendering history
2025-04-28 20:33:04,093 INFO    MainThread:19329 [wandb_run.py:_footer_history_summary_info():4192] rendering summary
2025-04-28 20:33:04,094 INFO    MainThread:19329 [wandb_run.py:_footer_sync_info():4121] logging synced files
